src/main/java/baritone/behavior/MemoryBehavior.java

```diff
@@ -167,6 +167,10 @@ public void onBlockInteract(BlockInteractEvent event) {
     @Override
     public void onPlayerDeath() {
         baritone.getWorldProvider().getCurrentWorld().getWaypoints().addWaypoint(new Waypoint("death", Waypoint.Tag.DEATH, ctx.playerFeet()));
+        if (baritone.settings().stopOnDeath.value) {
+            baritone.getPathingBehavior().cancelEverything();
+            System.out.println("stopOnDeath is set, cancelling all tasks due to death");
```

Comment:
shouldn't this be using baritone's logger, not `System.out`?

Reply:
seems that all the other messages in the file are using `System.out`... I guess I can change it

Reply:
Nevermind, that's fine then I guess.

Reply:
too late, I force pushed. do you want me to undo?